app/models/category_default_timer.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+class CategoryDefaultTimer < BaseTimer
+  belongs_to :user
+  belongs_to :category, foreign_key: :timerable_id
+  belongs_to :publishing_category, class_name: "Category", foreign_key: :category_id
+
+  validates :user_id, presence: true
+  validates :timerable_id, presence: true
+  validates :status_type, uniqueness: { scope: %i[timerable_id deleted_at] }, if: :public_type?
```

Comment:
```suggestion
  validates :status_type, uniqueness: { scope: %i[timerable_id] }, if: :public_type?
```

This should just be the above. It looks like `BaseTimer` includes `Trashable` but I think we only need it for the `TopicTimer` model only and not `CategoryDefaultTimer`.

Reply:
Can we exclude `Trashable` from `BaseTimer` and only include it in `TopicTimer`? `CategoryDefaultTimer` does not need to be [Trashable](https://github.com/discourse/discourse/blob/ac554f1f60b95b403c1015899ffa384296cdef46/app/models/concerns/trashable.rb).